mod/logrotate/start.php

```diff
@@ -64,28 +51,37 @@ function logrotate_archive_cron($hook, $entity_type, $returnvalue, $params) {
 function logrotate_delete_cron($hook, $entity_type, $returnvalue, $params) {
 	$resulttext = elgg_echo("logrotate:logdeleted");
 
-	$day = 86400;
-
-	$offset = 0;
 	$period = elgg_get_plugin_setting('delete', 'logrotate');
+	$offset = getSecondsInPeriod($period);
+
+	if (!log_browser_delete_log($offset)) {
+		$resulttext = elgg_echo("logrotate:lognotdeleted");
+	}
+
+	return $returnvalue . $resulttext;
+}
+
+
+/**
+ * @param $period
+ * @return int
+ */
+function getSecondsInPeriod($period)
+{
+	$secondsInDay = 86400;
```

Comment:
`$seconds_in_day`. We only use camelCase for class methods, snake_case for everything else.